game/src/Game/Autoload/Events.gd

```diff
@@ -33,13 +34,26 @@ func _ready():
 		compatibility_mode_path = arg_dictionary.get(&"compatibility-mode", compatibility_mode_path)
 
 	var start := Time.get_ticks_usec()
-
+	
+	GameDebug = load("res://src/Game/Autoload/Events/GameDebug.gd").new()
+	loading_screen.update_loading_screen(15)
+	Options = load("res://src/Game/Autoload/Events/Options.gd").new()
+	loading_screen.update_loading_screen(25)
+	Localisation = load("res://src/Game/Autoload/Events/Localisation.gd").new()
+	loading_screen.update_loading_screen(45)
+	ShaderManager = load("res://src/Game/Autoload/Events/ShaderManager.gd").new()
+	loading_screen.update_loading_screen(50, true)
```

Comment:
```suggestion
	GameDebug = preload("res://src/Game/Autoload/Events/GameDebug.gd").new()
	loading_screen.update_loading_screen(15)
	Options = preload("res://src/Game/Autoload/Events/Options.gd").new()
	loading_screen.update_loading_screen(25)
	Localisation = preload("res://src/Game/Autoload/Events/Localisation.gd").new()
	loading_screen.update_loading_screen(45)
	ShaderManager = preload("res://src/Game/Autoload/Events/ShaderManager.gd").new()
	loading_screen.update_loading_screen(50, true)
```

Since they're hardcoded right now, `preload` is preferable to `load`.